fem/fe/fe_pos.cpp

```diff
@@ -23,12 +24,17 @@ using namespace std;
 void PositiveFiniteElement::Project(
    Coefficient &coeff, ElementTransformation &Trans, Vector &dofs) const
 {
-   for (int i = 0; i < dof; i++)
-   {
-      const IntegrationPoint &ip = Nodes.IntPoint(i);
-      Trans.SetIntPoint(&ip);
-      dofs(i) = coeff.Eval(Trans, ip);
-   }
+   DenseMatrix mass;
```

Comment:
This change is more or less orthogonal to the purpose of this PR. However, a lot of my testing was predicated on needing exact projections of initial data using the Bernstein basis. But now I am a little bit stuck. If I remove this and address projection in a separate PR, the tests will fail. If I merge as-is, we at least temporarily lose the ability to use the original monotone interpolation operator. Another option is to merge both PR's into this one: the L2 proj for coarsening and the L2 projection for projection of coefficients.